src/Nethermind/Nethermind.Network/P2P/Subprotocols/Eth/V67/Eth67ProtocolHandler.cs

```diff
@@ -0,0 +1,65 @@
+﻿//  Copyright (c) 2021 Demerzel Solutions Limited
+//  This file is part of the Nethermind library.
+//
+//  The Nethermind library is free software: you can redistribute it and/or modify
+//  it under the terms of the GNU Lesser General Public License as published by
+//  the Free Software Foundation, either version 3 of the License, or
+//  (at your option) any later version.
+//
+//  The Nethermind library is distributed in the hope that it will be useful,
+//  but WITHOUT ANY WARRANTY; without even the implied warranty of
+//  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+//  GNU Lesser General Public License for more details.
+//
+//  You should have received a copy of the GNU Lesser General Public License
+//  along with the Nethermind. If not, see <http://www.gnu.org/licenses/>.
+//
+
+using Nethermind.Consensus;
+using Nethermind.Core.Specs;
+using Nethermind.Logging;
+using Nethermind.Network.P2P.Subprotocols.Eth.V65;
+using Nethermind.Network.P2P.Subprotocols.Eth.V66;
+using Nethermind.Network.Rlpx;
+using Nethermind.Stats;
+using Nethermind.Synchronization;
+using Nethermind.TxPool;
+
+namespace Nethermind.Network.P2P.Subprotocols.Eth.V67;
+
+/// <summary>
+/// https://github.com/ethereum/EIPs/blob/master/EIPS/eip-4938.md
+/// </summary>
+public class Eth67ProtocolHandler : Eth66ProtocolHandler
+{
+    public Eth67ProtocolHandler(ISession session,
+        IMessageSerializationService serializer,
+        INodeStatsManager nodeStatsManager,
+        ISyncServer syncServer,
+        ITxPool txPool,
+        IPooledTxsRequestor pooledTxsRequestor,
+        IGossipPolicy gossipPolicy,
+        ISpecProvider specProvider,
+        ILogManager logManager)
+        : base(session, serializer, nodeStatsManager, syncServer, txPool, pooledTxsRequestor, gossipPolicy, specProvider, logManager)
+    {
+    }
+
+    public override string Name => "eth67";
+
+    public override byte ProtocolVersion => 67;
+
+    public override void HandleMessage(ZeroPacket message)
+    {
+        switch (message.PacketType)
+        {
+            case Eth66MessageCode.GetNodeData:
+                break;
```

Comment:
So essentially, it will just timeout?

Reply:
hmm, good point, I will rethink it

Reply:
What does the spec says?

Reply:
Spec obligates us to remove message types `GetNodeData` and `NodeData` from the `eth` protocol, is not specifying our exact behaviour:
https://eips.ethereum.org/EIPS/eip-4938#specification
So I think we should do the same as we do when receiving other not supported messages - ignore without respond